Configurations/10-main.conf

```diff
@@ -934,28 +934,28 @@ sub vms_info {
     # in order to build "universal" binary and allow OpenSSL take
     # advantage of NEON when it's available.
     #
-    "android-armeabi" => {
-        inherit_from     => [ "android", asm("armv4_asm") ],
+    "android-armeabi-base" => {
+        inherit_from     => [ asm("armv4_asm") ],
```

Comment:
I feel this strange. Take android-mips-base for example:
```
    "android-mips-base" => {
        inherit_from     => [ asm("mips32_asm") ],
        template         => 1,
        perlasm_scheme   => "o32",
    },
```
I thought ```perlasm_scheme   => "o32"``` is an extra directive for ```asm("mips32_asm")```? Move asm("mips32_asm") to compiler-specific targets sounds strange. Or I should move ```perlasm_scheme``` as well?